docs/guides/composition-revisions-example.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Composition Revision Example
+weight: 101
+---
+This tutorial discusses how CompositionRevisions work and how they manage Composite Resource
+(XR) updates. This start with a `Composition` and `CompositeResourceDefinition` that defines a `MyVPC`
+resource.
+
+This tutorial creates multiple XRs and modifies Composition to apply different Composition Revisions to the XRs.
+
+## Install Crossplane
+1. Install Crossplane v1.11.0 or later.
+```shell
+kubectl create namespace crossplane-system
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm repo update
+helm install crossplane --namespace crossplane-system crossplane-master/crossplane --devel --version 1.11.0-rc.0.108.g0521c32e
+kubectl get pods -n crossplane-system
+```
+Expected Output:
+```shell
+NAME                                       READY   STATUS    RESTARTS   AGE
+crossplane-7f75ddcc46-f4d2z                1/1     Running   0          9s
+crossplane-rbac-manager-78bd597746-sdv6w   1/1     Running   0          9s
+```
+
+## Apply a Composition
+Apply the example Composition.
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  labels:
+    channel: dev
+  name: myvpcs.aws.example.upbound.io
+spec:
+  writeConnectionSecretsToNamespace: crossplane-system
+  compositeTypeRef:
+    apiVersion: aws.example.upbound.io/v1alpha1
+    kind: MyVPC
+  resources:
+  - base:
+      apiVersion: ec2.aws.upbound.io/v1beta1
+      kind: VPC
+      spec:
+        forProvider:
+          region: us-west-1
+          cidrBlock: 192.168.0.0/16
+          enableDnsSupport: true
+          enableDnsHostnames: true
+    name: my-vcp
+```
+
+
+## Apply a Composite Resource Definition
+
+Apply the example XRD.
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: myvpcs.aws.example.upbound.io
+spec:
+  group: aws.example.upbound.io
+  names:
+    kind: MyVPC
+    plural: myvpcs
+  versions:
+  - name: v1alpha1
+    served: true 
+    referenceable: true 
+    schema:
+      openAPIV3Schema:
+        type: object 
+        properties:
+          spec:
+            type: object 
+            properties:
+              id:
+                type: string 
+                description: ID of this VPC that other objects will use to refer to it. 
+            required:
+            - id
+```
+
+## Verify the Composition revision
+Verify that Crossplane created the Composition revision
+```shell
+kubectl get compositionrevisions -o=custom-columns=NAME:.metadata.name,REVISION:.spec.revision,LABEL:.metadata.labels.channel
+```
+Expected Output:
+```shell
+NAME                                    REVISION   LABEL
+myvpcs.aws.example.upbound.io-ad265bc   1          dev
+```
+
+The label `dev` is automatically created from the Composition.
```

Comment:
Should the column title be CHANNEL instead?

And with something like:
```suggestion
All labels propagated from `Composition` to the `CompositionRevision` including the `channel` label, which we printed as a column using the get query we built above.
```

Reply:
updated as CHANNEL